examples/osm_ped/cirros_2vnf_ns/cirros_2vnf_nsd.yaml

```diff
@@ -0,0 +1,27 @@
+nsd:nsd-catalog:
```

Comment:
Is this a ped? check https://github.com/CN-UPB/tng-sdk-benchmark/blob/master/examples/peds/ped_calibration_stress.yml